hs_core/templates/pages/metadata_terms.html

```diff
@@ -108,14 +108,28 @@
             <li><strong>Description: </strong> a text statement that points to a rights
                 management or usage statement for the resource.</li>
         </ul>
-
+        <a name="isCopiedFrom"></a>
         <ul>
             <li><strong>Term: </strong>URL</li>
             <li><strong>URI: </strong><a href="#URL">http://www.hydroshare.org/terms/URL</a></li>
             <li><strong>Label: </strong>Uniform Resource Locator</li>
             <li><strong>Description: </strong>URL that points to a rights
                 management or usage statement for the resource.</li>
         </ul>
+        <a name="isHostedBy"></a>
```

Comment:
I don't think it's a bug. Because of the header menu area, the anchor element position creates this problem. The revised positioning of the anchor element as you have done solves this problem. But it is still a problem for the first anchor element. For the time being you can do the repositioning of rest of the anchor elements. @mauriel133 Do you know a better way to resolve this?